paparazzi/src/test/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtensionShrinkTest.kt

```diff
@@ -0,0 +1,60 @@
+package app.cash.paparazzi.accessibility
+
+import android.widget.TextView
+import app.cash.paparazzi.DeviceConfig
+import app.cash.paparazzi.Paparazzi
+import app.cash.paparazzi.Snapshot
+import app.cash.paparazzi.SnapshotHandler
+import app.cash.paparazzi.internal.ImageUtils
+import com.android.ide.common.rendering.api.SessionParams
+import org.junit.Rule
+import org.junit.Test
+import java.awt.image.BufferedImage
+import java.io.File
+import javax.imageio.ImageIO
+
+class AccessibilityRenderExtensionShrinkTest {
```

Comment:
nit: can we combine this with `AccessibilityRenderExtensionTest` + TestParameterInjector?  Looks like `renderingMode` is the only argument that differs between the two.